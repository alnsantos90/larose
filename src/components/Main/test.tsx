import { render, screen } from '@testing-library/react'
import { Main } from '.'

describe('<Main />', () => {
  it('should render the heading', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /Boilerplate NextJs/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })

  it('should render the colors correctly', () => {
    const { container } = render(<Main />)

    expect(container.firstChild).toHaveClass('bg-gray-900')
  })
})
