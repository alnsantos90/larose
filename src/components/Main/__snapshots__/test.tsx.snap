// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<main
  class="bg-gray-900 text-white w-full h-[100vh] overflow-x-hidden overflow-y-hidden flex items-center flex-col justify-center"
>
  <div
    class="flex items-center"
  >
    <img
      alt="Alt Logo"
      class="w-[8rem] mx-5"
      src="assets/logo.svg"
    />
    <h1
      class="text-7xl my-6"
    >
      Boilerplate NextJs
    </h1>
  </div>
  <p
    class="text-4xl max-w-3xl text-center leading-relaxed"
  >
    Boilerplate utilizando NextJs, Tailwindcss, Jest, Husky, React Testing library
  </p>
</main>
`;
